oolab/src/main/java/agh/ics/oop/GrassField.java

```diff
@@ -0,0 +1,80 @@
+package agh.ics.oop;
+
+import org.w3c.dom.html.HTMLMapElement;
+
+import java.lang.reflect.Array;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+
+public class GrassField extends AbstractWorldMap{
+
+    private int fieldsNum;
+    private Vector2d[] positionsVect;
```

Comment:
to bym wyliczał każdorazowo przy pobieraniu szerokości/wysokości, inaczej trzeba to synchronizować przy przesuwaniu zwierzątek.